spanner/cloud-client/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -273,32 +273,36 @@ static void deleteExampleData(DatabaseClient dbClient) {
 
   // [START spanner_query_data]
   static void query(DatabaseClient dbClient) {
-    // singleUse() can be used to execute a single read or query against Cloud Spanner.
-    ResultSet resultSet =
+    // We use a try-with-resource block to automatically release resources held by ResultSet.
+    try (ResultSet resultSet =
         dbClient
+            // singleUse() can be used to execute a single read or query against Cloud Spanner.
             .singleUse()
-            .executeQuery(Statement.of("SELECT SingerId, AlbumId, AlbumTitle FROM Albums"));
-    while (resultSet.next()) {
-      System.out.printf(
-          "%d %d %s\n", resultSet.getLong(0), resultSet.getLong(1), resultSet.getString(2));
+            .executeQuery(Statement.of("SELECT SingerId, AlbumId, AlbumTitle FROM Albums"))) {
+      while (resultSet.next()) {
+        System.out.printf(
+            "%d %d %s\n", resultSet.getLong(0), resultSet.getLong(1), resultSet.getString(2));
+      }
     }
   }
   // [END spanner_query_data]
 
   // [START spanner_read_data]
   static void read(DatabaseClient dbClient) {
-    ResultSet resultSet =
+    // We use a try-with-resource block to automatically release resources held by ResultSet.
+    try (ResultSet resultSet =
         dbClient
             .singleUse()
             .read(
                 "Albums",
                 // KeySet.all() can be used to read all rows in a table. KeySet exposes other
```

Comment:
Again, can this be shortened to be on the same line as KeySet.All() ?

Reply:
Done.